.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: "es5",
+  tabWidth: 4,
+  semi: true,
+  singleQuote: false,
```

Comment:
I personally prefer tab width as 2 and use single quotes

I think we should change this, lot of space is wasted when tab width is 4, and every time we have to type double quotes we have to hit 2 keys, shift + quotes key, where as in single quotes we just hit one key

Reply:
@bhtibrewal the more common accepted way is single quotes.

also tabWitdth seems alright to me with 4 spaces

Reply:
Thank you @prakashchoudhary07 and @vvaibhavdesai for pointing out these, as you both and mahima said I have made the change singlequote: true and we are going ahead with tabwidth 4